tests/dot.cpp

```diff
@@ -134,6 +134,58 @@ namespace {
     //dot (fake_executor_t (), x, y, dotResultPar);
     //EXPECT_EQ( dotResultPar, expectedDotResult );
   }
+
+  TEST(BLAS1_dot, mdspan_complex_double_test2)
+  {
+    using real_t = double;
+    using scalar_t = std::complex<real_t>;
+    using vector_t = mdspan<scalar_t, extents<dynamic_extent>>;
+
+    constexpr ptrdiff_t vectorSize(5);
```

Comment:
Would you consider using `size_t` instead of `ptrdiff_t`, given that `mdspan` has changed to use `size_t` for extents?

Reply:
happy to help out: https://github.com/kokkos/stdBLAS/pull/104